events/open-source-marathon.html

```diff
@@ -31,6 +31,183 @@
         content: "-";
         width: 1.5rem;
       }
+/*styles of glitch effect*/
+      .glitch-wrapper {
+   width: 100%;
+   height: 100%;
+   display: flex;
+   align-items: center;
+   justify-content: center;
+   text-align: center;
+  
+}
+
+.glitch {
+   position: relative;
+   font-size: 3.75rem;
+   font-weight: bold;
+   color: #FFFFFF;
+   letter-spacing: 3px;
+   z-index: 1;
+}
+
+.glitch:before {
+   content: attr(data-text);
+   position: absolute;
+   top: 0;
+   left: -2px;
+   width: 100%;
+   color: 50px;
+   background-image: url("/assets/img/grain-dark.png");
+   overflow: hidden;
+   clip: rect(0, 900px, 0, 0);
+   animation: noise-before 2.5s infinite linear alternate-reverse;
+}
+
+.glitch:after {
+   content: attr(data-text);
+   position: absolute;
+   top: 0;
+   left: 2px;
+   width: 100%;
+   color: 50px;
+   background-image: url("/assets/img/grain-dark.png");
```

Comment:
Same applies here as well.

Reply:
color should be color:#ffffff;   yes we actually don't need backgroud-Image.   Should I again make pull request or is there ianother method to correct ?